NEWS.md

```diff
@@ -8,6 +8,7 @@
 ### Enhancements
 
 * Updated the documentation and vignettes with the new way of specifying data for `teal::init()`. The `data` argument accepts `teal_data` object.
+* Added `{logger}` functionality for logging changes in shiny inputs in all modules. `{logger}` was added to Imports.
```

Comment:
Why `{pkg_name}` instead of `pkg_name`? Please have a look at the conventions so far. If it was like that then let's leave it as is.

Reply:
no idea, I thought `{pkg_name}` was the convention but it's not :) will remove curly brackets while adding `if(shiny:isRunning()) {` around `logger::log_shiny_input_changes(`.